src/components/compose.jsx

```diff
@@ -604,7 +604,15 @@ function Compose({
         const item = items[i];
         if (item.kind === 'file') {
           const file = item.getAsFile();
-          if (file && supportedMimeTypes.includes(file.type)) {
+          if (!file) {
+            alert(`Could not access the given attachment.`);
```

Comment:
This would need the i18n `t`. The copy "Unable to attach file." would be better in my opinion.

Reply:
```suggestion
            alert(t`Unable to attach file.`);
```

Here you go!